README.md

```diff
@@ -1,7 +1,10 @@
-# TeamViewer AD Connector (PowerShell)
+# TeamViewer Active Directory Connector (AD-Connector)
 
 A simple PowerShell integration script to synchronize users of an Active
-Directory Group to a TeamViewer company.
+Directory (AD) group to a TeamViewer company via REST based API.
+
+Further information can be found here:
+https://community.teamviewer.com/t5/Knowledge-Base/Active-Directory-Connector-AD-Connector/ta-p/31158
```

Comment:
It might make sense to have a different text for the link, as it is pretty long:
Something like: "AD-Connector Knowledge Base Article".
Can be done like this
```
[AD-Connector Knowledge Base Article](https://community.teamviewer.com/t5/Knowledge-Base/Active-Directory-Connector-AD-Connector/ta-p/31158)
```